spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/DefaultSource.scala

```diff
@@ -0,0 +1,147 @@
+package org.apache.spark.sql.cassandra
+
+import org.apache.spark.sql.{DataFrame, SaveMode, SQLContext}
+import org.apache.spark.sql.SaveMode._
+import org.apache.spark.sql.sources.{CreatableRelationProvider, SchemaRelationProvider, BaseRelation, RelationProvider}
+import org.apache.spark.sql.types.{DataType, StructType}
+
+/**
+ * Cassandra data source extends [[RelationProvider]], [[SchemaRelationProvider]] and [[CreatableRelationProvider]].
```

Comment:
No need to repeat implemented interfaces in ScalaDoc. They are visible in the signature.
ScalaDoc should document what's not visible in the signature.